assets/scss/layouts/products/_productSaleBadges.scss

```diff
@@ -0,0 +1,112 @@
+// =============================================================================
+// PRODUCT Sale Badges (CSS)
+// =============================================================================
+
+// BURST BADGE
+// -----------------------------------------------------------------------------
+.sale-flag-star,
+.sale-flag-star:before,
+.sale-flag-star:after {
+    background: stencilColor("color_badge_product_sale_badges");
+    display: block;
+    float: none;
+    height: 50px;
+    overflow: visible;
+    position: relative;
+    transform: scaleX(1) scaleY(1) scaleZ(1);
+    transform-origin: 50% 50% 0;
+    width: 50px;
+    z-index: auto;
+}
+
+.sale-flag-star:before {
+    content: "";
+    left: 0;
+    position: absolute;
+    top: 0;
+    transform: rotateZ(30deg) scaleX(1) scaleY(1) scaleZ(1);
+}
+
+.sale-flag-star:after {
+    content: "";
+    left: 0;
+    position: absolute;
+    top: 0;
+    transform: rotateZ(60deg) scaleX(1) scaleY(1) scaleZ(1);
+}
+
+.starwrap {
+    height: 50px;
+    position: absolute;
+    top: 10px;
+    width: 50px;
+}
+
+span.sale-text-burst {
+    color: stencilColor("color_text_product_sale_badges");
+    font-weight: 600;
+    line-height: 1rem;
+    position: absolute;
+    text-align: center;
+    top: 20%;
+    width: 100%;
+    z-index: 999;
```

Comment:
We try to stay away from arbitrary values for z-index so that they don't interfere with other parts of the UI, e.g. Modals, flyouts. Can you use something from [our z-index map](https://github.com/bigcommerce/citadel/blob/fc1958d9f5731e228d02926a207f8cc393947c5b/src/settings/global/z-index/_z-index.scss) instead?